pkg/ddevapp/drupal.go

```diff
@@ -41,6 +42,7 @@ func NewDrupalSettings() *DrupalSettings {
 		DatabasePrefix:   "",
 		HashSalt:         util.RandString(64),
 		Signature:        DdevFileSignature,
+		SyncDir:          "sites/default/files/sync",
```

Comment:
This isn't correct, and results in loads of trouble. This should be relative to the docroot.